apps/indexer/src/api/controller/token-distribution/controller.ts

```diff
@@ -15,6 +15,10 @@ export function tokenDistribution(
   app: Hono,
   repository: TokenDistributionRepository,
 ) {
+  for (let i = 0; i < 10; i++) {
```

Comment:
The loop with `console.log(i);` appears to be a temporary debug statement. Please remove or replace it with proper logging before merging to maintain clean and professional code. Additionally, consider adding meaningful comments or extracting this logic into a dedicated method if it serves a specific purpose.